claat/fetch/drive/auth/auth.go

```diff
@@ -43,7 +47,9 @@ var (
 		ClientID:     googClient,
 		ClientSecret: googSecret,
 		Scopes:       []string{scopeDriveReadOnly},
-		RedirectURL:  "urn:ietf:wg:oauth:2.0:oob",
+		// RedirectURL:  "urn:ietf:wg:oauth:2.0:oob",
+		// RedirectURL:  "https://qwiklab-production.appspot.com/oauth2callback",
+		RedirectURL:  "http://localhost:8080",
```

Comment:
Did you want to keep this change?

Reply:
yeah, when switching the creds over to qwiklabs-services-prod I first tried as a "Desktop Application" (which must be the way Codelabs Preview works) but the auth screen threw an error. As a Web app, it was allowed but could only get access to the auth code with the localhost redirect.

Reply:
Okay then. This whole thing is still kinda weird and I'm even more confused how this auth flow works now.